docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1209,49 +1206,163 @@ visual inspection and quantitatively using an appropriate statistical
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 99 (inclusive) to use as
+   the next two digits.
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 9999 (inclusive) to use as
+   the last four digits.
+
+ITIN generation
+'''''''''''''''
+
+**ITINs should be unique**.
+The same ITIN should not be assigned to two different simulants.
+
+ITIN generation is similar to SSN generation, using these steps:
+
+#. Generate a zero-padded integer between **900 and 999** (inclusive) to use as
+   the first three digits.
+#. Append a dash.
+#. Generate a zero-padded integer between **50 and 65, 70 and 88, 90 and 92, or 94 and 99** (inclusive) to use as
+   the next two digits.
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 9999 (inclusive) to use as
+   the last four digits.
+
+Slides from the IRS on the ITIN format can be found here: https://www.irs.gov/pub/irs-pdf/p4757.pdf
+
+Population initialization
+'''''''''''''''''''''''''
+
+At population initialization, we follow these rules to initialize an SSN or ITIN for every simulant:
+
+#. If the ACS person sampled was born in the United States (according to the :code:`NATIVITY` column),
+   the new simulant is assigned an SSN.
+#. Otherwise, the simulant is assigned an SSN **74.3%** of the time, and an ITIN the remainder of the time.
+
+Fertility
+'''''''''
+
+When a simulant is born during the sim to a parent who lives in the United States,
+they are **always** assigned an SSN.
+
+If a simulant is born during the sim to a parent who lives outside the United States,
+they are assigned an SSN **if and only if** their parent has one.
+If their parent has an ITIN (i.e. does not have an SSN), the newborn is assigned an ITIN.
+
+Immigration
+'''''''''''
+
+When new simulants are created via immigration into the US, we follow these rules to initialize an SSN or ITIN for every simulant:
+
+#. If the ACS person sampled was born in the United States (according to the :code:`NATIVITY` column),
+   the new simulant is assigned an SSN.
+#. Otherwise, the simulant is assigned an SSN **62.5%** of the time, and an ITIN the remainder of the time.
+
+Limitations
+^^^^^^^^^^^
+
+#. We assume that all people born in the US have an SSN.
+   In reality, some people, especially the very old, may not have one.
+   We think this is a minor issue.
+#. We do not correlate having an SSN with any other characteristics, e.g. demographics or
+   location, among the foreign-born population.
+#. We calculated SSN assignment percentages at population initialization and immigration
+   using the assumption that all documented immigrants to the US have an SSN, and all undocumented
+   immigrants do not.
+#. We do not allow those with ITINs to switch to SSNs during their life, which can happen in real life
+   at e.g. naturalization.
+#. We never expire ITINs; in real life they expire after three years of non-use.
+   This means tax-to-tax matching with ITINs may be unrealistically easy.
```

Comment:
I am fine with this for now, but I do think it might be a PRL miss for us in the first iteration

Reply:
Yes -- I briefly thought about trying to expire them, but I think it is a bit hard to do without duplicating a lot of tax observer logic.

Reply:
Yeah, we would have to track taxes for 3 years and see if they are observed to then assign a new number. That would be overly complex right now